scripts/src/main/resources/scripts/command/java

```diff
@@ -91,8 +87,214 @@ function doCreate() {
       cd "${DEVON_IDE_HOME}/workspaces/main" || exit 255
     fi
   fi
-  doMavenArchetype "${@}"
-  exit ${?}
+
+local framework
+local mavenArgs
+local package
+local artifactId
+while [[ -n "${1}" ]]
+do
+  if [[ "${1}" == "-f" ]] || [[ "${1}" == "-framework" ]]  
+  then 
+    if [[ "${2,,}" == "quarkus" ]]
+    then 
+      framework="quarkus"
+      shift
+    elif [[ "${2,,}" == "spring" ]]
+    then 
+      framework="spring"
+      shift
+    fi
+  elif [[ "${1:0:8}" == "-Ddbtype" ]]
+  then 
+    dbtype=${1:9}
+  elif [[ "${1:0:2}" == "-D" ]]
+  then
+      mavenargs="${mavenargs} ${1}"
+  elif [[ "${1}" =~ [a-z]+[.][a-z]+([.][a-z]+)* ]]
+  then 
+      package="${package} ${1}"
+  elif [[ "${1}" =~ [^.]* ]]
+  then
+    ## some better definition for a artifactID TODO
+      artifactId+=${1}
+  else
+      doFail "Wrong parameter for -f"
+  fi
+shift
+done
+
+local getFramework=""
```

Comment:
should be `framework` instead of `getFramework`, right?
Isnt shellcheck detecting this?

Reply:
mhm, actually `local framework` has already been defined in 91.
I do not see why you need another variable for it.

Reply:
I removed the duplicate code for framework detection. So it's also better readable.